vncviewer/DesktopWindow.cxx

```diff
@@ -51,6 +51,7 @@
 
 #ifdef __APPLE__
 #include "cocoa.h"
+#include <CoreGraphics/CoreGraphics.h>
```

Comment:
gave up trying to support individual headers and am now importing the whole thing. _maybe_ this will alleviate the compile errors.